nixos/lib/testing-python.nix

```diff
@@ -10,6 +10,9 @@
 , extraConfigurations ? [ ]
 }:
 
+# TODO(jsoo1): Drop the commit that adds this line. This line was all
+# that remains of a commit in our fork. I don't think it is necessary.
+# with import ./build-vms.nix { inherit system pkgs minimal extraConfigurations specialArgs; };
```

Comment:
I'm pretty sure you can drop this.  If I remember correctly the only reason for this line was for back when we used `awakeShared` via `specialArgs`.  Also, even if you were to uncomment the line it would do nothing

Reply:
Ok! I'll drop it.  Thank you!